ui-v2/app/components/feedback-dialog/index.js

```diff
@@ -9,11 +9,11 @@ const STATE_ERROR = 'error';
 export default Component.extend(SlotsMixin, {
   wait: service('timeout'),
   dom: service('dom'),
-  classNames: ['with-feedback'],
   transition: '',
   transitionClassName: 'feedback-dialog-out',
   state: STATE_READY,
   permanent: true,
+  tagName: '',
```

Comment:
@johncowen The test failing is caused by the addition of this `tagName`. I don't understand why. Can you please take a look?

Reply:
(just commenting for history here!) Ah sorry I missed that we were using `this.element` in here. Giving a component an empty `tagName` means you can no longer use `this.element` as a reference, there are various replacements for this, one of them being the `{{ref}}` modifier which we use here.